src/lib/api.js

```diff
@@ -32,6 +32,15 @@ app.use((req, res, next) => {
     next();
 });
 
+/**
+ * Middleware that converts uncaught Errors to failureJson
+ */
+app.use((err, req, res, next) => {
+    if (err) {
+        res.failureJson(err.message);
```

Comment:
NACK: I don't think we should respond with unexpected error messages. Not that this API will actually be publicly-facing, but responding with unexpected, verbose error messages can be a security issue. We should probably respond with a generic message such as "Unexpected internal server error".